Cython/Plex/Transitions.py

```diff
@@ -119,8 +113,7 @@ def iteritems(self,
 
     # ------------------- Private methods --------------------
```

Comment:
in a separate pass it might be worth making these private with underscores

Reply:
As long as the tests pass, it should be fine to rename them.